### PR TITLE
enabler: change registration and administration service to use config from env

### PR DESCRIPTION
**Pull request by evegufy**
change registration and administration service to use config from env
fix: remove trailing slash for bpdm url

### CODE REVIEW

File: charts/portal/templates/deployment-backend-administration.yaml [hunk 1]
```diff
@@ -40,13 +40,294 @@ spec:
...
+              key: "central-db-password"
+        - name: "DATABASEACCESS__KEYCLOAK__CONNECTIONSTRING"
+          value: "Server={{ .Values.backend.keycloak.central.dbConnection.host }};Database={{ .Values.backend.keycloak.central.dbConnection.database }};Port={{ .Values.backend.keycloak.central.dbConnection.port }};User Id={{ .Values.backend.keycloak.central.dbConnection.user }};Password=$(KEYCLOAK_CENTRAL_PASSWORD);Ssl Mode={{ .Values.backend.keycloak.central.dbConnection.sslMode }};"
+        - name: "DATABASEACCESS__KEYCLOAK__DATABASESCHEMA"
+          value: "{{ .Values.backend.keycloak.central.dbConnection.schema }}"
+        - name: "DATABASEACCESS__PORTAL__CONNECTIONSTRING"
+          value: "Server={{ .Values.postgresql.fullnameOverride }}-primary;Database={{ .Values.postgresql.auth.database }};Port={{ .Values.backend.dbConnection.port }};User Id={{ .Values.postgresql.auth.portalUser }};Password=$(PORTAL_PASSWORD);Ssl Mode={{ .Values.backend.dbConnection.sslMode }};"
+        - name: "DATABASEACCESS__PORTAL__DATABASESCHEMA"
+          value: "{{ .Values.backend.dbConnection.schema }}"
+        - name: "DEBUGENABLED"
```
Comment (Phil91) [inline charts/portal/templates/deployment-backend-administration.yaml:190]: my suggestion would be to get rid of this configuration in all our services and use a check to the environment instead

File: charts/portal/values-preprod.yaml [hunk 2]
```diff
@@ -216,7 +225,7 @@ backend:
       custodianLibrary: "Debug"
       sdfactoryLibrary: "Debug"
     bpdm:
-      baseAddress: "https://portal-gate.pre-prod.demo.catena-x.net/"
+      baseAddress: "https://portal-gate.pre-prod.demo.catena-x.net"
```
Comment (Phil91) [inline charts/portal/values-preprod.yaml:228]: we need to adjust the code for that to include the / in the get, put request. will include this in the portal [PR](https://github.com/catenax-ng/tx-portal-backend/pull/164)

**Review by Phil91 — APPROVED** [review]

File: charts/portal/values-preprod.yaml (hunk 2 above)
Comment (evegufy) [reply charts/portal/values-preprod.yaml:222]: great, thank you!

File: charts/portal/templates/deployment-backend-administration.yaml (hunk 1 above)
Comment (evegufy) [reply charts/portal/templates/deployment-backend-administration.yaml]: done

**Pull request merged**